agent.py

```diff
@@ -62,7 +62,7 @@
 # Constants
 PID_NAME = "sts-agent"
 PID_DIR = None
-WATCHDOG_MULTIPLIER = 100
+WATCHDOG_MULTIPLIER = 10
```

Comment:
@vishalxebia looks good, just should just be 100 as the default now. Same with the config below

Reply:
the default was 10 always and 100 just for KPN, we wanted to do. so it's good if we keep last time default.